aptos-move/block-executor/src/view.rs

```diff
@@ -229,6 +257,30 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> LatestView<
         }
         ret
     }
+
+    /// Given a state value, performs deserialization-serialization round-trip
+    /// to replace any aggregator / aggregator snapshot values.
+    fn lift_aggregators_v2_from_state_value(
+        &self,
+        state_value: StateValue,
+        layout: &MoveTypeLayout,
```

Comment:
I'm not sure. Should this be Option<MoveTypeLayout>?

Reply:
This is only called if we have Some(layout), so having concrete reference here is fine